[2023-12-05T19:32:39.754+0000] {processor.py:157} INFO - Started process (PID=36123) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:32:39.755+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:32:39.757+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:32:39.756+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:32:39.776+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:32:39.773+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/plugins/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T19:32:39.777+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:32:39.805+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.056 seconds
[2023-12-05T19:33:10.480+0000] {processor.py:157} INFO - Started process (PID=36179) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:33:10.482+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:33:10.483+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:33:10.483+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:33:10.494+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:33:10.492+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/plugins/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T19:33:10.495+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:33:10.516+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.039 seconds
[2023-12-05T19:33:40.998+0000] {processor.py:157} INFO - Started process (PID=36235) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:33:40.999+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:33:41.001+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:33:41.000+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:33:41.013+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:33:41.010+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/plugins/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T19:33:41.014+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:33:41.042+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.048 seconds
[2023-12-05T19:34:11.368+0000] {processor.py:157} INFO - Started process (PID=36292) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:34:11.369+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:34:11.371+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:34:11.370+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:34:11.381+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:34:11.379+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/plugins/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T19:34:11.382+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:34:11.403+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.039 seconds
[2023-12-05T19:34:41.675+0000] {processor.py:157} INFO - Started process (PID=36348) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:34:41.677+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:34:41.678+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:34:41.678+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:34:41.690+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:34:41.688+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/plugins/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T19:34:41.692+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:34:41.715+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.044 seconds
[2023-12-05T19:34:42.824+0000] {processor.py:157} INFO - Started process (PID=36352) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:34:42.826+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:34:42.827+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:34:42.827+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:34:42.842+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:34:42.839+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/plugins/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T19:34:42.843+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:34:42.871+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.051 seconds
[2023-12-05T19:35:12.969+0000] {processor.py:157} INFO - Started process (PID=36405) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:35:12.970+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:35:12.971+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:35:12.971+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:35:12.983+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:35:12.980+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/plugins/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T19:35:12.984+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:35:13.008+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.044 seconds
[2023-12-05T19:36:14.496+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:36:14.511+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:36:14.518+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:36:14.517+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:36:14.539+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:36:14.532+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
ImportError: cannot import name 'FileCheckOperator' from 'foobar_operator' (/opt/airflow/plugins/foobar_operator.py)
[2023-12-05T19:36:14.542+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:36:14.572+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.085 seconds
[2023-12-05T19:36:44.752+0000] {processor.py:157} INFO - Started process (PID=94) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:36:44.754+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:36:44.757+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:36:44.757+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:36:44.764+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:36:44.763+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
ImportError: cannot import name 'FileCheckOperator' from 'foobar_operator' (/opt/airflow/plugins/foobar_operator.py)
[2023-12-05T19:36:44.766+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:36:44.790+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.042 seconds
[2023-12-05T19:36:59.944+0000] {processor.py:157} INFO - Started process (PID=143) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:36:59.945+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:36:59.948+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:36:59.948+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:36:59.959+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:36:59.957+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:36:59.960+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:36:59.990+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.051 seconds
[2023-12-05T19:37:30.462+0000] {processor.py:157} INFO - Started process (PID=199) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:37:30.463+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:37:30.465+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:37:30.464+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:37:30.472+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:37:30.470+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:37:30.473+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:37:30.496+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.039 seconds
[2023-12-05T19:38:00.838+0000] {processor.py:157} INFO - Started process (PID=255) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:38:00.840+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:38:00.842+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:38:00.841+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:38:00.849+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:38:00.847+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:38:00.850+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:38:00.877+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.044 seconds
[2023-12-05T19:38:31.082+0000] {processor.py:157} INFO - Started process (PID=311) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:38:31.084+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:38:31.085+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:38:31.085+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:38:31.093+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:38:31.091+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:38:31.094+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:38:31.118+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.041 seconds
[2023-12-05T19:39:01.311+0000] {processor.py:157} INFO - Started process (PID=367) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:39:01.313+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:39:01.314+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:39:01.314+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:39:01.322+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:39:01.320+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:39:01.322+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:39:01.345+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.040 seconds
[2023-12-05T19:39:31.449+0000] {processor.py:157} INFO - Started process (PID=423) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:39:31.451+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:39:31.452+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:39:31.452+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:39:31.459+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:39:31.458+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:39:31.460+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:39:31.483+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.039 seconds
[2023-12-05T19:40:01.956+0000] {processor.py:157} INFO - Started process (PID=479) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:40:01.958+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:40:01.959+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:40:01.959+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:40:01.966+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:40:01.964+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:40:01.967+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:40:01.991+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.039 seconds
[2023-12-05T19:40:32.218+0000] {processor.py:157} INFO - Started process (PID=535) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:40:32.220+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:40:32.221+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:40:32.221+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:40:32.230+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:40:32.228+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:40:32.231+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:40:32.262+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.054 seconds
[2023-12-05T19:41:02.772+0000] {processor.py:157} INFO - Started process (PID=598) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:41:02.774+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:41:02.775+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:41:02.775+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:41:02.784+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:41:02.782+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:41:02.785+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:41:02.810+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.043 seconds
[2023-12-05T19:41:32.960+0000] {processor.py:157} INFO - Started process (PID=654) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:41:32.962+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:41:32.964+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:41:32.963+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:41:32.970+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:41:32.969+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:41:32.971+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:41:32.992+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.036 seconds
[2023-12-05T19:45:00.575+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:45:00.576+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:45:00.580+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:45:00.580+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:45:00.587+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:45:00.585+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:45:00.588+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:45:00.617+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.048 seconds
[2023-12-05T19:45:30.823+0000] {processor.py:157} INFO - Started process (PID=94) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:45:30.825+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:45:30.828+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:45:30.828+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:45:30.835+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:45:30.833+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:45:30.836+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:45:30.856+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.037 seconds
[2023-12-05T19:49:34.581+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:49:34.582+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:49:34.586+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:49:34.585+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:49:34.595+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:49:34.592+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:49:34.596+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:49:34.626+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.050 seconds
[2023-12-05T19:50:05.178+0000] {processor.py:157} INFO - Started process (PID=94) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:50:05.180+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:50:05.183+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:50:05.183+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:50:05.190+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:50:05.188+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:50:05.191+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:50:05.214+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.040 seconds
[2023-12-05T19:50:35.417+0000] {processor.py:157} INFO - Started process (PID=150) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:50:35.419+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:50:35.422+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:50:35.421+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:50:35.429+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:50:35.427+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:50:35.430+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:50:35.454+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.041 seconds
[2023-12-05T19:51:05.872+0000] {processor.py:157} INFO - Started process (PID=206) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:51:05.874+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:51:05.877+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:51:05.876+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:51:05.883+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:51:05.882+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:51:05.884+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:51:05.907+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.039 seconds
[2023-12-05T19:51:36.251+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:51:36.269+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:51:36.274+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:51:36.273+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:51:36.284+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:51:36.282+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:51:36.286+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:51:36.321+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.076 seconds
[2023-12-05T19:52:33.916+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:52:33.917+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:52:33.921+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:52:33.921+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:52:33.931+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:52:33.928+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:52:33.933+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:52:33.963+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.052 seconds
[2023-12-05T19:53:04.691+0000] {processor.py:157} INFO - Started process (PID=94) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:53:04.692+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:53:04.695+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:53:04.695+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:53:04.702+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:53:04.701+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:53:04.703+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:53:04.728+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.042 seconds
[2023-12-05T19:53:34.937+0000] {processor.py:157} INFO - Started process (PID=150) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:53:34.938+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:53:34.941+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:53:34.941+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:53:34.948+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:53:34.947+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:53:34.949+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:53:34.974+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.042 seconds
[2023-12-05T19:54:05.180+0000] {processor.py:157} INFO - Started process (PID=206) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:54:05.181+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:54:05.184+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:54:05.184+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:54:05.191+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:54:05.190+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:54:05.192+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:54:05.218+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.043 seconds
[2023-12-05T19:54:35.548+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:54:35.549+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:54:35.553+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:54:35.553+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:54:35.560+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:54:35.558+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:54:35.561+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:54:35.585+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.041 seconds
[2023-12-05T19:55:05.720+0000] {processor.py:157} INFO - Started process (PID=318) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:55:05.722+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:55:05.726+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:55:05.725+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:55:05.733+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:55:05.731+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:55:05.733+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:55:05.761+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.046 seconds
[2023-12-05T19:57:22.458+0000] {processor.py:157} INFO - Started process (PID=46) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:57:22.461+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:57:22.473+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:57:22.472+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:57:22.533+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:57:22.527+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:57:22.545+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:57:22.599+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.148 seconds
[2023-12-05T19:57:52.943+0000] {processor.py:157} INFO - Started process (PID=102) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:57:52.945+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:57:52.963+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:57:52.963+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:57:52.990+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:57:52.985+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:57:52.991+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:57:53.031+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.114 seconds
[2023-12-05T19:58:23.172+0000] {processor.py:157} INFO - Started process (PID=158) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:58:23.174+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:58:23.178+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:58:23.177+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:58:23.186+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:58:23.184+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:58:23.187+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:58:23.323+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.156 seconds
[2023-12-05T19:58:53.731+0000] {processor.py:157} INFO - Started process (PID=214) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:58:53.733+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:58:53.740+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:58:53.738+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:58:53.753+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:58:53.750+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:58:53.755+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:58:53.787+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.061 seconds
[2023-12-05T19:59:23.959+0000] {processor.py:157} INFO - Started process (PID=270) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:59:23.961+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:59:23.965+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:59:23.964+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:59:23.982+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:59:23.978+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:59:23.983+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:59:24.026+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.071 seconds
[2023-12-05T19:59:54.843+0000] {processor.py:157} INFO - Started process (PID=326) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:59:54.850+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T19:59:54.864+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:59:54.862+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:59:54.912+0000] {logging_mixin.py:154} INFO - [2023-12-05T19:59:54.903+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T19:59:54.915+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T19:59:55.005+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.179 seconds
[2023-12-05T20:00:25.101+0000] {processor.py:157} INFO - Started process (PID=382) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:00:25.105+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:00:25.110+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:00:25.109+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:00:25.130+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:00:25.125+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:00:25.132+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:00:25.192+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.099 seconds
[2023-12-05T20:00:55.799+0000] {processor.py:157} INFO - Started process (PID=438) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:00:55.800+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:00:55.803+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:00:55.803+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:00:55.811+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:00:55.809+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:00:55.812+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:00:55.845+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.052 seconds
[2023-12-05T20:01:25.965+0000] {processor.py:157} INFO - Started process (PID=494) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:01:25.967+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:01:25.972+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:01:25.971+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:01:25.981+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:01:25.979+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:01:25.982+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:01:26.011+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.050 seconds
[2023-12-05T20:01:56.049+0000] {processor.py:157} INFO - Started process (PID=550) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:01:56.051+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:01:56.054+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:01:56.054+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:01:56.060+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:01:56.059+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:01:56.061+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:01:56.082+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.037 seconds
[2023-12-05T20:02:26.554+0000] {processor.py:157} INFO - Started process (PID=606) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:02:26.555+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:02:26.558+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:02:26.558+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:02:26.565+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:02:26.563+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:02:26.566+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:02:26.591+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.042 seconds
[2023-12-05T20:02:57.008+0000] {processor.py:157} INFO - Started process (PID=662) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:02:57.009+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:02:57.015+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:02:57.014+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:02:57.024+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:02:57.022+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:02:57.025+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:02:57.052+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.049 seconds
[2023-12-05T20:03:27.539+0000] {processor.py:157} INFO - Started process (PID=718) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:03:27.540+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:03:27.543+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:03:27.543+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:03:27.550+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:03:27.549+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:03:27.551+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:03:27.577+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.043 seconds
[2023-12-05T20:03:57.920+0000] {processor.py:157} INFO - Started process (PID=774) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:03:57.922+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:03:57.925+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:03:57.925+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:03:57.933+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:03:57.931+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:03:57.934+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:03:57.958+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.043 seconds
[2023-12-05T20:04:28.192+0000] {processor.py:157} INFO - Started process (PID=830) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:04:28.193+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:04:28.197+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:04:28.197+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:04:28.205+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:04:28.203+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:04:28.206+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:04:28.237+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.051 seconds
[2023-12-05T20:04:58.756+0000] {processor.py:157} INFO - Started process (PID=886) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:04:58.757+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:04:58.761+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:04:58.760+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:04:58.768+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:04:58.766+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:04:58.769+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:04:58.794+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.042 seconds
[2023-12-05T20:05:29.071+0000] {processor.py:157} INFO - Started process (PID=942) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:05:29.072+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:05:29.075+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:05:29.075+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:05:29.082+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:05:29.081+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:05:29.083+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:05:29.107+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.040 seconds
[2023-12-05T20:06:44.899+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:06:44.901+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:06:44.904+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:06:44.904+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:06:44.913+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:06:44.911+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:06:44.915+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:06:44.943+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.051 seconds
[2023-12-05T20:08:17.088+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:08:17.089+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:08:17.097+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:08:17.097+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:08:17.104+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:08:17.102+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:08:17.107+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:08:17.135+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.053 seconds
[2023-12-05T20:08:48.016+0000] {processor.py:157} INFO - Started process (PID=94) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:08:48.018+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:08:48.021+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:08:48.021+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:08:48.027+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:08:48.026+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from ..plugins.foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:08:48.028+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:08:48.053+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.042 seconds
[2023-12-05T20:14:58.164+0000] {processor.py:157} INFO - Started process (PID=47) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:14:58.186+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:14:58.190+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:14:58.190+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:14:58.212+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:14:58.208+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:14:58.213+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:14:58.458+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.309 seconds
[2023-12-05T20:15:28.684+0000] {processor.py:157} INFO - Started process (PID=103) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:15:28.685+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:15:28.688+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:15:28.687+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:15:28.695+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:15:28.694+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:15:28.696+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:15:28.722+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.046 seconds
[2023-12-05T20:15:59.122+0000] {processor.py:157} INFO - Started process (PID=159) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:15:59.124+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:15:59.130+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:15:59.129+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:15:59.150+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:15:59.144+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:15:59.152+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:15:59.200+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.085 seconds
[2023-12-05T20:16:29.650+0000] {processor.py:157} INFO - Started process (PID=215) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:16:29.651+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:16:29.654+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:16:29.654+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:16:29.661+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:16:29.659+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from .foobar_operator import FileCheckOperator
ImportError: attempted relative import with no known parent package
[2023-12-05T20:16:29.661+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:16:29.680+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.034 seconds
[2023-12-05T20:16:48.349+0000] {processor.py:157} INFO - Started process (PID=264) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:16:48.351+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:16:48.357+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:16:48.357+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:16:48.407+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:16:48.401+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:16:48.411+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:16:48.455+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.112 seconds
[2023-12-05T20:17:18.811+0000] {processor.py:157} INFO - Started process (PID=320) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:17:18.812+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:17:18.816+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:17:18.815+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:17:18.829+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:17:18.825+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:17:18.831+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:17:18.857+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.051 seconds
[2023-12-05T20:17:48.951+0000] {processor.py:157} INFO - Started process (PID=376) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:17:48.952+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:17:48.959+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:17:48.958+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:17:48.973+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:17:48.970+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:17:48.975+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:17:49.023+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.078 seconds
[2023-12-05T20:18:19.433+0000] {processor.py:157} INFO - Started process (PID=432) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:18:19.434+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:18:19.436+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:18:19.436+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:18:19.444+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:18:19.442+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:18:19.445+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:18:19.463+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.034 seconds
[2023-12-05T20:18:50.133+0000] {processor.py:157} INFO - Started process (PID=488) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:18:50.135+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:18:50.140+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:18:50.139+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:18:50.154+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:18:50.151+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:18:50.156+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:18:50.196+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.070 seconds
[2023-12-05T20:19:20.548+0000] {processor.py:157} INFO - Started process (PID=544) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:19:20.550+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:19:20.554+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:19:20.553+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:19:20.600+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:19:20.591+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:19:20.606+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:19:20.643+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.101 seconds
[2023-12-05T20:19:51.192+0000] {processor.py:157} INFO - Started process (PID=598) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:19:51.193+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:19:51.195+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:19:51.195+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:19:51.203+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:19:51.202+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:19:51.204+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:19:51.224+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.036 seconds
[2023-12-05T20:20:21.701+0000] {processor.py:157} INFO - Started process (PID=654) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:20:21.703+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:20:21.707+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:20:21.706+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:20:21.716+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:20:21.714+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:20:21.717+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:20:21.742+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.045 seconds
[2023-12-05T20:20:52.437+0000] {processor.py:157} INFO - Started process (PID=710) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:20:52.439+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:20:52.443+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:20:52.443+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:20:52.453+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:20:52.451+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:20:52.455+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:20:52.485+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.054 seconds
[2023-12-05T20:21:53.765+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:21:53.767+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:21:53.771+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:21:53.771+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:21:53.784+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:21:53.781+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:21:53.786+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:21:53.817+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.061 seconds
[2023-12-05T20:22:24.586+0000] {processor.py:157} INFO - Started process (PID=94) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:22:24.587+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:22:24.590+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:22:24.589+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:22:24.598+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:22:24.596+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:22:24.599+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:22:24.620+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.038 seconds
[2023-12-05T20:22:55.153+0000] {processor.py:157} INFO - Started process (PID=150) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:22:55.155+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:22:55.160+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:22:55.159+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:22:55.172+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:22:55.169+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:22:55.173+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:22:55.201+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.052 seconds
[2023-12-05T20:23:25.447+0000] {processor.py:157} INFO - Started process (PID=206) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:23:25.448+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:23:25.451+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:23:25.451+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:23:25.461+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:23:25.459+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:23:25.463+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:23:25.488+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.046 seconds
[2023-12-05T20:23:55.683+0000] {processor.py:157} INFO - Started process (PID=261) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:23:55.684+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:23:55.688+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:23:55.687+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:23:55.698+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:23:55.695+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:23:55.699+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:23:55.724+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.045 seconds
[2023-12-05T20:24:26.280+0000] {processor.py:157} INFO - Started process (PID=317) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:24:26.282+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:24:26.287+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:24:26.286+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:24:26.303+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:24:26.300+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:24:26.305+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:24:26.333+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.059 seconds
[2023-12-05T20:24:56.522+0000] {processor.py:157} INFO - Started process (PID=373) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:24:56.524+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:24:56.527+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:24:56.527+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:24:56.537+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:24:56.534+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:24:56.538+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:24:56.563+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.048 seconds
[2023-12-05T20:25:26.699+0000] {processor.py:157} INFO - Started process (PID=429) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:25:26.701+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:25:26.704+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:25:26.704+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:25:26.714+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:25:26.712+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:25:26.715+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:25:26.741+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.046 seconds
[2023-12-05T20:25:57.303+0000] {processor.py:157} INFO - Started process (PID=485) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:25:57.305+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:25:57.309+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:25:57.308+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:25:57.319+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:25:57.316+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:25:57.320+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:25:57.344+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.045 seconds
[2023-12-05T20:26:27.664+0000] {processor.py:157} INFO - Started process (PID=541) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:26:27.665+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:26:27.668+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:26:27.668+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:26:27.678+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:26:27.676+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:26:27.679+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:26:27.702+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.042 seconds
[2023-12-05T20:26:58.016+0000] {processor.py:157} INFO - Started process (PID=597) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:26:58.017+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:26:58.021+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:26:58.020+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:26:58.031+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:26:58.028+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:26:58.032+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:26:58.057+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.045 seconds
[2023-12-05T20:27:28.402+0000] {processor.py:157} INFO - Started process (PID=653) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:27:28.404+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:27:28.408+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:27:28.407+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:27:28.418+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:27:28.416+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:27:28.419+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:27:28.442+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.045 seconds
[2023-12-05T20:27:58.573+0000] {processor.py:157} INFO - Started process (PID=709) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:27:58.574+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:27:58.578+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:27:58.577+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:27:58.590+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:27:58.586+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:27:58.591+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:27:58.616+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.047 seconds
[2023-12-05T20:28:28.842+0000] {processor.py:157} INFO - Started process (PID=765) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:28:28.843+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:28:28.846+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:28:28.846+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:28:28.856+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:28:28.854+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:28:28.857+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:28:28.882+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.045 seconds
[2023-12-05T20:28:59.304+0000] {processor.py:157} INFO - Started process (PID=821) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:28:59.306+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:28:59.309+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:28:59.309+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:28:59.319+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:28:59.317+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:28:59.320+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:28:59.344+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.045 seconds
[2023-12-05T20:29:29.616+0000] {processor.py:157} INFO - Started process (PID=877) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:29:29.618+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:29:29.621+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:29:29.621+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:29:29.630+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:29:29.628+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:29:29.631+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:29:29.655+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.043 seconds
[2023-12-05T20:30:00.080+0000] {processor.py:157} INFO - Started process (PID=933) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:30:00.082+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:30:00.085+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:30:00.084+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:30:00.094+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:30:00.091+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:30:00.095+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:30:00.121+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.044 seconds
[2023-12-05T20:30:30.307+0000] {processor.py:157} INFO - Started process (PID=989) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:30:30.308+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:30:30.312+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:30:30.311+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:30:30.322+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:30:30.319+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:30:30.323+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:30:30.347+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.044 seconds
[2023-12-05T20:31:00.575+0000] {processor.py:157} INFO - Started process (PID=1045) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:31:00.576+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:31:00.579+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:31:00.579+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:31:00.588+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:31:00.586+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:31:00.589+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:31:00.613+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.043 seconds
[2023-12-05T20:31:30.873+0000] {processor.py:157} INFO - Started process (PID=1101) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:31:30.874+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:31:30.877+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:31:30.877+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:31:30.886+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:31:30.884+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:31:30.887+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:31:30.911+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.043 seconds
[2023-12-05T20:32:01.271+0000] {processor.py:157} INFO - Started process (PID=1157) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:32:01.272+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:32:01.276+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:32:01.276+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:32:01.285+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:32:01.283+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:32:01.286+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:32:01.309+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.043 seconds
[2023-12-05T20:32:31.729+0000] {processor.py:157} INFO - Started process (PID=1213) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:32:31.731+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:32:31.734+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:32:31.734+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:32:31.744+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:32:31.742+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:32:31.745+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:32:31.770+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.045 seconds
[2023-12-05T20:33:01.938+0000] {processor.py:157} INFO - Started process (PID=1269) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:33:01.939+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:33:01.943+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:33:01.943+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:33:01.952+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:33:01.950+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:33:01.954+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:33:01.978+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.044 seconds
[2023-12-05T20:33:32.150+0000] {processor.py:157} INFO - Started process (PID=1325) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:33:32.152+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:33:32.156+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:33:32.156+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:33:32.166+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:33:32.164+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:33:32.168+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:33:32.196+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.050 seconds
[2023-12-05T20:34:02.594+0000] {processor.py:157} INFO - Started process (PID=1381) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:34:02.595+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:34:02.598+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:34:02.598+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:34:02.607+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:34:02.605+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from foobar_operator import FileCheckOperator
  File "/opt/airflow/dags/foobar_operator.py", line 14, in <module>
    from .foobar_trigger import FileCheckTrigger
ImportError: attempted relative import with no known parent package
[2023-12-05T20:34:02.608+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:34:02.630+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.040 seconds
[2023-12-05T20:34:27.136+0000] {processor.py:157} INFO - Started process (PID=1437) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:34:27.138+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:34:27.141+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:34:27.141+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:34:27.151+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:34:27.149+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:34:27.152+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:34:27.177+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.045 seconds
[2023-12-05T20:34:57.641+0000] {processor.py:157} INFO - Started process (PID=1493) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:34:57.642+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:34:57.645+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:34:57.645+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:34:57.653+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:34:57.652+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:34:57.654+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:34:57.677+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.041 seconds
[2023-12-05T20:35:27.965+0000] {processor.py:157} INFO - Started process (PID=1549) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:35:27.967+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:35:27.970+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:35:27.969+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:35:27.977+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:35:27.975+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:35:27.978+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:35:28.002+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.041 seconds
[2023-12-05T20:35:58.501+0000] {processor.py:157} INFO - Started process (PID=1605) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:35:58.503+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:35:58.506+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:35:58.506+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:35:58.514+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:35:58.512+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:35:58.514+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:35:58.538+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.041 seconds
[2023-12-05T20:36:28.856+0000] {processor.py:157} INFO - Started process (PID=1659) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:36:28.857+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:36:28.860+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:36:28.860+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:36:28.868+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:36:28.866+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:36:28.869+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:36:28.893+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.041 seconds
[2023-12-05T20:36:59.219+0000] {processor.py:157} INFO - Started process (PID=1715) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:36:59.221+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:36:59.224+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:36:59.223+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:36:59.231+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:36:59.229+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:36:59.232+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:36:59.254+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.039 seconds
[2023-12-05T20:37:29.817+0000] {processor.py:157} INFO - Started process (PID=1771) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:37:29.819+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:37:29.822+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:37:29.822+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:37:29.830+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:37:29.828+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:37:29.831+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:37:29.852+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.039 seconds
[2023-12-05T20:38:00.682+0000] {processor.py:157} INFO - Started process (PID=1827) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:38:00.683+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:38:00.687+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:38:00.686+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:38:00.695+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:38:00.693+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:38:00.696+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:38:00.718+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.040 seconds
[2023-12-05T20:38:31.586+0000] {processor.py:157} INFO - Started process (PID=1884) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:38:31.588+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:38:31.591+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:38:31.590+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:38:31.598+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:38:31.596+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:38:31.599+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:38:31.621+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.039 seconds
[2023-12-05T20:39:02.370+0000] {processor.py:157} INFO - Started process (PID=1940) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:39:02.371+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:39:02.374+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:39:02.374+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:39:02.381+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:39:02.380+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:39:02.383+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:39:02.409+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.042 seconds
[2023-12-05T20:39:33.013+0000] {processor.py:157} INFO - Started process (PID=1996) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:39:33.015+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:39:33.018+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:39:33.018+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:39:33.026+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:39:33.024+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/a_foobar_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/a_foobar_dag.py", line 6, in <module>
    from airflow.  foobar_operator import FileCheckOperator
ModuleNotFoundError: No module named 'airflow.foobar_operator'
[2023-12-05T20:39:33.027+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:39:33.049+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.040 seconds
[2023-12-05T20:40:02.274+0000] {processor.py:157} INFO - Started process (PID=2052) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:40:02.275+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:40:02.278+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:02.278+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:40:02.530+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:40:02.650+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:02.650+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-12-05T20:40:02.662+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:02.661+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-12-05T20:40:02.669+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:02.669+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-12-05T20:40:02.670+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:02.670+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:40:02.683+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:02.682+0000] {dag.py:2963} INFO - Creating ORM DAG for a_foobar_dag
[2023-12-05T20:40:02.684+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:02.683+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:40:02.703+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.433 seconds
[2023-12-05T20:40:33.104+0000] {processor.py:157} INFO - Started process (PID=2108) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:40:33.106+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:40:33.109+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:33.108+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:40:33.314+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:40:33.334+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:33.334+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:40:33.348+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:40:33.348+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:40:33.368+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.268 seconds
[2023-12-05T20:41:03.607+0000] {processor.py:157} INFO - Started process (PID=2164) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:41:03.609+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:41:03.612+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:41:03.611+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:41:03.678+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:41:03.702+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:41:03.702+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:41:03.718+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:41:03.718+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:41:03.742+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.138 seconds
[2023-12-05T20:41:33.943+0000] {processor.py:157} INFO - Started process (PID=2220) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:41:33.945+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:41:33.948+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:41:33.947+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:41:34.018+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:41:34.041+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:41:34.041+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:41:34.057+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:41:34.057+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:41:34.080+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.141 seconds
[2023-12-05T20:42:29.239+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:42:29.243+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:42:29.251+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:42:29.250+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:42:29.280+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:42:29.483+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:42:29.483+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:42:29.502+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:42:29.502+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:42:29.531+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.300 seconds
[2023-12-05T20:42:59.614+0000] {processor.py:157} INFO - Started process (PID=94) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:42:59.615+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:42:59.619+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:42:59.618+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:42:59.632+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:42:59.660+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:42:59.660+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:42:59.680+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:42:59.679+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:42:59.703+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.094 seconds
[2023-12-05T20:43:30.131+0000] {processor.py:157} INFO - Started process (PID=150) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:43:30.132+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:43:30.135+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:43:30.135+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:43:30.148+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:43:30.173+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:43:30.173+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:43:30.191+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:43:30.190+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:43:30.218+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.092 seconds
[2023-12-05T20:44:00.688+0000] {processor.py:157} INFO - Started process (PID=206) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:44:00.690+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:44:00.693+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:44:00.692+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:44:00.706+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:44:00.731+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:44:00.731+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:44:00.748+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:44:00.748+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:44:00.770+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.086 seconds
[2023-12-05T20:44:31.144+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:44:31.146+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:44:31.149+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:44:31.148+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:44:31.162+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:44:31.187+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:44:31.186+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:44:31.204+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:44:31.204+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:44:31.229+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.089 seconds
[2023-12-05T20:45:01.403+0000] {processor.py:157} INFO - Started process (PID=318) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:45:01.405+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:45:01.411+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:45:01.410+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:45:01.430+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:45:01.470+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:45:01.469+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:45:01.498+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:45:01.498+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:45:01.535+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.138 seconds
[2023-12-05T20:45:31.721+0000] {processor.py:157} INFO - Started process (PID=375) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:45:31.723+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:45:31.727+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:45:31.726+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:45:31.741+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:45:31.767+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:45:31.766+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:45:31.780+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:45:31.780+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:45:31.803+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.085 seconds
[2023-12-05T20:46:02.119+0000] {processor.py:157} INFO - Started process (PID=431) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:46:02.121+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:46:02.125+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:46:02.125+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:46:02.139+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:46:02.167+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:46:02.166+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:46:02.187+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:46:02.187+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:46:02.246+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.131 seconds
[2023-12-05T20:46:32.388+0000] {processor.py:157} INFO - Started process (PID=487) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:46:32.405+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:46:32.409+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:46:32.408+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:46:32.425+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:46:32.453+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:46:32.453+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:46:32.470+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:46:32.470+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:46:32.494+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.111 seconds
[2023-12-05T20:47:02.691+0000] {processor.py:157} INFO - Started process (PID=543) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:47:02.693+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:47:02.697+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:47:02.697+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:47:02.716+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:47:02.747+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:47:02.747+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:47:02.765+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:47:02.765+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:47:02.793+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.107 seconds
[2023-12-05T20:47:33.264+0000] {processor.py:157} INFO - Started process (PID=599) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:47:33.265+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:47:33.268+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:47:33.268+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:47:33.285+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:47:33.320+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:47:33.320+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:47:33.340+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:47:33.340+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:47:33.376+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.117 seconds
[2023-12-05T20:48:35.516+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:48:35.517+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:48:35.520+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:48:35.520+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:48:35.539+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:48:35.749+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:48:35.749+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:48:35.770+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:48:35.769+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:48:35.797+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.287 seconds
[2023-12-05T20:49:05.918+0000] {processor.py:157} INFO - Started process (PID=94) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:49:05.920+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:49:05.923+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:49:05.923+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:49:05.937+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:49:05.964+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:49:05.963+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:49:05.981+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:49:05.981+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:49:06.006+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.092 seconds
[2023-12-05T20:49:36.375+0000] {processor.py:157} INFO - Started process (PID=150) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:49:36.377+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:49:36.380+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:49:36.380+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:49:36.394+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:49:36.419+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:49:36.419+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:49:36.437+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:49:36.437+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:49:36.461+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.091 seconds
[2023-12-05T20:50:06.913+0000] {processor.py:157} INFO - Started process (PID=206) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:50:06.916+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:50:06.921+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:50:06.920+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:50:06.964+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:50:07.016+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:50:07.015+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:50:07.040+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:50:07.040+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:50:07.071+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.163 seconds
[2023-12-05T20:50:37.364+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:50:37.366+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:50:37.369+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:50:37.369+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:50:37.385+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:50:37.412+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:50:37.412+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:50:37.430+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:50:37.430+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:50:37.456+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.097 seconds
[2023-12-05T20:51:08.004+0000] {processor.py:157} INFO - Started process (PID=318) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:51:08.006+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:51:08.010+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:51:08.009+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:51:08.023+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:51:08.049+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:51:08.048+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:51:08.066+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:51:08.065+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:51:08.092+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.093 seconds
[2023-12-05T20:51:38.170+0000] {processor.py:157} INFO - Started process (PID=374) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:51:38.171+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:51:38.174+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:51:38.174+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:51:38.188+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:51:38.220+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:51:38.219+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:51:38.239+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:51:38.239+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:51:38.269+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.104 seconds
[2023-12-05T20:52:08.400+0000] {processor.py:157} INFO - Started process (PID=430) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:52:08.402+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:52:08.406+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:52:08.406+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:52:08.420+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:52:08.449+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:52:08.449+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:52:08.469+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:52:08.468+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:52:08.496+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.100 seconds
[2023-12-05T20:52:38.751+0000] {processor.py:157} INFO - Started process (PID=486) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:52:38.753+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:52:38.756+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:52:38.756+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:52:38.771+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:52:38.797+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:52:38.796+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:52:38.814+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:52:38.814+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:52:38.839+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.092 seconds
[2023-12-05T20:53:09.494+0000] {processor.py:157} INFO - Started process (PID=542) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:53:09.496+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:53:09.499+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:53:09.499+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:53:09.512+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:53:09.538+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:53:09.537+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:53:09.555+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:53:09.554+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:53:09.578+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.088 seconds
[2023-12-05T20:53:40.041+0000] {processor.py:157} INFO - Started process (PID=598) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:53:40.042+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:53:40.045+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:53:40.045+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:53:40.060+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:53:40.089+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:53:40.088+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:53:40.108+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:53:40.108+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:53:40.135+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.099 seconds
[2023-12-05T20:54:10.220+0000] {processor.py:157} INFO - Started process (PID=654) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:54:10.221+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:54:10.226+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:54:10.225+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:54:10.240+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:54:10.270+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:54:10.269+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:54:10.289+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:54:10.289+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:54:10.322+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.107 seconds
[2023-12-05T20:54:40.567+0000] {processor.py:157} INFO - Started process (PID=711) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:54:40.569+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:54:40.573+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:54:40.572+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:54:40.586+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:54:40.612+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:54:40.612+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:54:40.631+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:54:40.631+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:54:40.657+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.094 seconds
[2023-12-05T20:55:10.822+0000] {processor.py:157} INFO - Started process (PID=767) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:55:10.823+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:55:10.827+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:55:10.827+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:55:10.841+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:55:10.868+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:55:10.868+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:55:10.886+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:55:10.886+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:55:10.912+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.095 seconds
[2023-12-05T20:55:41.367+0000] {processor.py:157} INFO - Started process (PID=823) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:55:41.368+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:55:41.371+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:55:41.371+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:55:41.385+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:55:41.411+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:55:41.411+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:55:41.430+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:55:41.430+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:55:41.454+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.091 seconds
[2023-12-05T20:56:11.690+0000] {processor.py:157} INFO - Started process (PID=879) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:56:11.692+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:56:11.695+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:56:11.694+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:56:11.709+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:56:11.736+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:56:11.736+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:56:11.754+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:56:11.754+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:56:11.783+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.098 seconds
[2023-12-05T20:56:41.833+0000] {processor.py:157} INFO - Started process (PID=935) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:56:41.834+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:56:41.838+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:56:41.838+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:56:41.852+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:56:41.879+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:56:41.879+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:56:41.897+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:56:41.897+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:56:41.922+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.094 seconds
[2023-12-05T20:57:12.312+0000] {processor.py:157} INFO - Started process (PID=991) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:57:12.313+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:57:12.316+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:57:12.316+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:57:12.329+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:57:12.354+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:57:12.354+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:57:12.371+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:57:12.370+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:57:12.394+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.087 seconds
[2023-12-05T20:57:42.824+0000] {processor.py:157} INFO - Started process (PID=1047) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:57:42.826+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:57:42.829+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:57:42.829+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:57:42.842+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:57:42.872+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:57:42.872+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:57:42.890+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:57:42.890+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:57:42.918+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.098 seconds
[2023-12-05T20:58:13.215+0000] {processor.py:157} INFO - Started process (PID=1103) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:58:13.217+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:58:13.221+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:58:13.220+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:58:13.234+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:58:13.261+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:58:13.261+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:58:13.280+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:58:13.279+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:58:13.305+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.094 seconds
[2023-12-05T20:58:43.394+0000] {processor.py:157} INFO - Started process (PID=1159) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:58:43.395+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:58:43.399+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:58:43.398+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:58:43.411+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:58:43.433+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:58:43.433+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:58:43.449+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:58:43.449+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:58:43.470+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.080 seconds
[2023-12-05T20:59:47.243+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:59:47.255+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T20:59:47.260+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:59:47.259+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:59:47.280+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T20:59:47.456+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:59:47.456+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T20:59:47.471+0000] {logging_mixin.py:154} INFO - [2023-12-05T20:59:47.471+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T20:59:47.498+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.261 seconds
[2023-12-05T21:00:17.882+0000] {processor.py:157} INFO - Started process (PID=94) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:00:17.883+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T21:00:17.886+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:00:17.886+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:00:17.899+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:00:17.926+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:00:17.925+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T21:00:17.943+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:00:17.942+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T21:00:17.965+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.088 seconds
[2023-12-05T21:00:48.244+0000] {processor.py:157} INFO - Started process (PID=150) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:00:48.246+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T21:00:48.249+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:00:48.249+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:00:48.265+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:00:48.293+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:00:48.292+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T21:00:48.310+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:00:48.310+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T21:00:48.335+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.097 seconds
[2023-12-05T21:01:18.538+0000] {processor.py:157} INFO - Started process (PID=206) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:01:18.539+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T21:01:18.542+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:01:18.542+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:01:18.555+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:01:18.586+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:01:18.585+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T21:01:18.603+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:01:18.603+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T21:01:18.633+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.100 seconds
[2023-12-05T21:01:48.741+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:01:48.742+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T21:01:48.746+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:01:48.745+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:01:48.758+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:01:48.783+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:01:48.783+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T21:01:48.800+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:01:48.800+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T21:01:48.824+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.088 seconds
[2023-12-05T21:02:19.020+0000] {processor.py:157} INFO - Started process (PID=326) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:02:19.021+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T21:02:19.025+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:02:19.024+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:02:19.038+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:02:19.066+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:02:19.065+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T21:02:19.084+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:02:19.084+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T21:02:19.108+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.093 seconds
[2023-12-05T21:02:49.477+0000] {processor.py:157} INFO - Started process (PID=382) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:02:49.479+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T21:02:49.483+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:02:49.483+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:02:49.495+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:02:49.517+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:02:49.517+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T21:02:49.535+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:02:49.535+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T21:02:49.556+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.085 seconds
[2023-12-05T21:03:19.849+0000] {processor.py:157} INFO - Started process (PID=438) to work on /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:03:19.851+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/a_foobar_dag.py for tasks to queue
[2023-12-05T21:03:19.857+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:03:19.856+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:03:19.880+0000] {processor.py:839} INFO - DAG(s) dict_keys(['a_foobar_dag']) retrieved from /opt/airflow/dags/a_foobar_dag.py
[2023-12-05T21:03:19.929+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:03:19.928+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-05T21:03:19.972+0000] {logging_mixin.py:154} INFO - [2023-12-05T21:03:19.972+0000] {dag.py:3722} INFO - Setting next_dagrun for a_foobar_dag to None, run_after=None
[2023-12-05T21:03:20.025+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/a_foobar_dag.py took 0.184 seconds
